backend/utility/emailTemplates.js

```diff
@@ -55,3 +56,13 @@ module.exports.ContactUsMailTemplate = (adminName, req) => {
   `;
   return emailContent;
 };
+
+module.exports.ContactUsCronJobMailTemplate = (adminName) => {
+  const emailContent = `
+  <h4>Hi ${adminName},<br>
+  Contact Us Data has been removed !
```

Comment:
Please change it to `The Contact Us Form data of last 2 month has been removed!`